File: CoreJava/man-boy-app/src/com/techlabs/empolyee/Developer.java
package com.techlabs.empolyee;

public class Developer extends Employee{

	private double perks;
	private double sal;
	public Developer(int e, String n, double b) {
		super(e, n, b);
	}

	@Override
	public double calNetSalary() {
		perks=basic*0.3;
		sal=basic+perks;
		return sal;
	}
	

}

File: C#/OOP/InheritanceExample/InheritanceExample/Developer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InheritanceExample
{
    class Developer : Employee
    {
        private double perks;
	private double sal;
	public Developer(int e, String n, double b) :
		base(e, n, b){
	}

	
	public override double calNetSalary() {
		perks=basic*0.3;
		sal=basic+perks;
		return sal;
	}
    }
}
